src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -19,13 +20,14 @@ import scala.util.{Failure, Success}
   */
 object Lorre extends App with LazyLogging {
 
-  //keep this import here to make it evident where we spawn our async code
-  implicit val system: ActorSystem = ActorSystem("lorre-system")
-  implicit val dispatcher = system.dispatcher
-
-  //how long to wait for graceful shutdown of system components
-  private[this] val shutdownWait = 10.seconds
+  final case class CombinedConfiguration(
+    lorre: LorreConfiguration,
+    tezos: TezosConfiguration,
```

Comment:
Would this have to be generalized the second we add another blockchain?

Reply:
It depends... as long as we have different classes/packages for separate networks, they will require separate custom configurations too.

Once we find a globally shared configuration fitting different chains, it will be appropriate to rename these configurations as well

Reply:
Since adding a new chain is not negligible work, I assume that updating the configuration objects will not be the pain point, but just one additional detail in the total cost of the operation